pkg/controller/sriovnetworknodepolicy/sriovnetworknodepolicy_controller.go

```diff
@@ -548,9 +548,10 @@ func renderDevicePluginConfigData(pl *sriovnetworkv1.SriovNetworkNodePolicyList)
 		}
 
 		found, i := resourceNameInList(p.Spec.ResourceName, &rcl)
+		netDeviceSelectors := dptypes.NetDeviceSelectors{}
```

Comment:
Could you explain why making following changes? Is it related to the sdk upgrade? If not, maybe it's better to put it in a separate PR.